packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -47,54 +47,355 @@ const tar = require('tar')
 const semver = require('semver')
 const slugify = require('slugify')
 const generatorPkg = require('../package.json')
+const Handlebars = require('handlebars')
```

Comment:
did we already have `.hbs` files... do we need to add this to 3PP filed deps with security? eek 😅

Reply:
That's a relief, do we even nee d a new one then?

Reply:
We do not have to create another 3PP request as long as this particular version is in the All 3PP report, which it is, [here](https://gus.lightning.force.com/lightning/r/ADM_Third_Party_Software__c/a0qEE00000023VLYAY/view)